### ISSUE TITLE
import statement unresolved causing error

### ISSUE THREAD

**Issue by MattSanford** [vapor/leaf#210]
### Describe the bug

 `#import("content")`  is failing to resolve causing an error `import("content") should have been resolved BEFORE serialization`

#### settings.leaf
```
#extend("base"):
    #export("content"):
    <p>Hello from settings</p>
    #endexport
#endextend
```

#### base.leaf
```
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8" />
  <title>#(title)</title>
</head>
<body>
#import("content")
</body>
</html>
```

#### Routes.swift
```swift
    app.get("settings") { req -> EventLoopFuture<View> in
        let context = SettingsContext.mocked
        return req.view.render("base", context)
    }
```

#### Package.swift
```swift
...
    dependencies: [
        // Dependencies declare other packages that this package depends on.
        .package(url: "https://github.com/vapor/vapor.git", from: "4.39.2"),
        .package(url: "https://github.com/vapor/leaf.git", from: "4.0.0"),
    ],
...

### Expected behavior
I expected to see "hello from Settings".

**Comment by 0xTim** [comment]
In your routes you're trying to render the base template instead of settings.

It should be
```swift
return req.view.render("settings", context)
```

**Comment by MattSanford** [comment]
Tim, you are a blessing 🙏🏼MattOn Jul 21, 2022, at 3:52 AM, Tim Condon ***@***.***> wrote:﻿
Closed #210 as completed.
